### PR TITLE
1.1.1a

### LINKED ISSUE #29: openssl 1.1.1 considered newer than 1.1.1a by conda
conda obeys PEP440 regarding versioning.  OpenSSL does not.  They start from 1.1.1 and 1.1.1a is the next version.  Conda thinks 1.1.1a is a lower version.

If 1.1.1 is available, conda will prefer it to 1.1.1a.  For defaults, we removed 1.1.1 so that 1.1.1a would get picked up.  Long-term, we will special-case openssl in conda.

@isuruf recommended potentially using epoch instead:

```
Isuru Fernando @isuruf 15:24
@msarahan, why not increase the epoch instead of removing 1.1.1?

Mike Sarahan @msarahan 15:25
what would the spec look like? I'm just not used to using epochs for anything

Isuru Fernando @isuruf 15:25
version = 1!1.1.1a
1.1.1 = 0!1.1.1 < 1!1.1.1a
```

### LINKED ISSUE #32: policy for version release
We should probably agree on a version release plan here. Currently we are on the 1.0.2 series, whereas also 1.1.0 and 1.1.1. have been released. 

How should we handle releases here? Since the build requirements seems to change from 1.0.2 to 1.1.1, probably it would be best to maintain a branch for each series.

How should we handle #29 across these branches? We could probably use some hacky epoch trick, but ~I'm not sure what is best (_i.e._ like `1020!1.0.2p`, `1021!1.0.2q`?)~ at the moment I can't imagine how.

Let me know you thoughts!

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #29, closes #32

### CODE REVIEW

**Comment by carlodri** [comment]
did you inspect what was going wrong in  $SRC_DIR/testsuite.log?

**Comment by hmaarrfk** [comment]
not yet, it is too late here for me think clearly. obviously that make command was working correctly....

**Comment by hmaarrfk** [comment]
(the one I changed that is)

**Comment by carlodri** [comment]
for inspecting we could simply make `make test` output to console rather than to a log file, what do you think?

**Comment by hmaarrfk** [comment]
lets try one more time. they had different capitalization for grep.

**Comment by carlodri** [comment]
great work @hmaarrfk!

**Comment by carlodri** [comment]
for the other @conda-forge/openssl maintainers coming here: I have created a `1.0.2` branch in the feedstock, which will stay there for future updates to the 1.0.2 series (see #32), whereas this PR would be merged into `master`.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,28 +8,32 @@ package:
...
   fn: {{ name }}-{{ version }}.tar.gz
   url: http://www.openssl.org/source/{{ name }}-{{ version }}.tar.gz
-  sha256: 50a98e07b1a89eb8f6a99477f262df71c6fa7bef77df4dc83025a2845c827d00
+  sha256: fc20130f8b7cbd2fb918b2f14e2f429e109c31ddd0fb38fc5d71d9ffed3f9f41
 
 build:
-  number: 1002
-  no_link: lib/libcrypto.so.1.0.0        # [linux]
-  no_link: lib/libcrypto.1.0.0.dylib     # [osx]
+  number: 0
```
Comment (carlodri) [inline recipe/meta.yaml]: @hmaarrfk should we bump back to 1000, rather than 0?

```suggestion
  number: 1000
```

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender preparing to merge

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender this should be ready to go

**Pull request merged**